unit_tests/dummy_remote.py

```diff
@@ -85,12 +85,20 @@ def get_db_auth():
     def is_kubernetes():
         return False
 
+    @staticmethod
+    def replace_dc_placeholders_with_dc_names(cmd):
+        datacenter_names = ['datacenter1']
+        for idx, dc in enumerate(datacenter_names):
+            cmd = cmd.replace(f"<dc{idx}>", dc)
```

Comment:
Maybe we can also handle the case
Of placeholder wasn't set ?

And replace the whole setting with the correct layout ? Or at least fail the test notifying us the actual datacenter isn't aligned with the c-s command ?

Reply:
I'm not sure if I understand you.
Generally, I wanted to keep possibility to hardcode (for whatever the reason) and also possibility to still use SimpleStrategy.
Also I thought of replacing all c-s commands in the repo which use NetworkStrategy to apply there placehloders. 

In case we don't want to ever use hardcoded datacenters, we could add the check at sct_config level - fail in case NetworkTopology and hardcoded datacenter names.